### ISSUE TITLE
issue connecting over HTTPS using CA

### ISSUE THREAD

**Issue by msterckx** [bparry02/ansible-tower-hashicorp-vault#1]
Hi,
Did you try to get this working using a HTTPS connection to access vault?
regards,
Michel

**Comment by bparry02** [comment]
No, sorry, I never tried that. I imagine it would work as long as the CA that signed the vault certificate was part of the host's ca-trust chain, or possibly if Tower allows a way to add a CA chain at runtime.
